### PR TITLE
Migrate boards systemd units to kano-peripherals

### PR DESCRIPTION
**Pull request by tombettany**
Systemd: Migrate boards unit files into kano-peripherals

The unit files which control the board daemon and the speaker LEDs
service should be owned by the package that provides those services.
Migrate so that this repository has no knowledge of them.

Also solves:

Debian: Add back compilation when packaging

When adding the i18n tasks, the `debian/rules` file was modified to
override the default build in order to build the i18n files; this
removed building of the default targets and hence caused the package to
fail to build. Add these back to restore building.

### CODE REVIEW

**Comment by tombettany** [comment]
[(3) As a user, I want the CK2 hat LEDs to visualize when the computer is working hard [5]](https://trello.com/c/Zmv1Vyp5/767-3-as-a-user-i-want-the-ck2-hat-leds-to-visualize-when-the-computer-is-working-hard-5)

File: debian/rules [hunk 1]
```diff
@@ -4,5 +4,6 @@
 	dh $@
 
 override_dh_auto_build:
+	make
```
Comment (radujipa) [inline debian/rules]: What should this address here?

Comment (tombettany) [inline debian/rules]: `dh` by default runs `make` during its `dh_auto_build` phase and it so happens that we do have a `Makefile` in the repo's root so it was actually doing something.

This [commit](https://github.com/KanoComputing/kano-desktop/commit/ecc23791ad89ed8e244fe33fd77b873e6f9aaa65) introduced the

```make
override_dh_auto_build:
	cd po && make messages.pot
	cd po && make
```

which completely overrides the default behaviour and hence the `Makefile` in the root is never run and hence the `kano-mount-trigger` executable is never compiled and so fails to be installed.

TL;DR The package fails to build without it.

Comment (radujipa) [inline debian/rules]: That makes sense, nice one 👍
